provider/aws/aws.go

```diff
@@ -585,8 +623,9 @@ func (p *AWSProvider) submitChanges(ctx context.Context, changes Route53Changes,
 
 				successfulChanges := 0
 
-				if _, err := p.client.ChangeResourceRecordSetsWithContext(ctx, params); err != nil {
-					log.Errorf("Failure in zone %s [Id: %s] when submitting change batch: %v", aws.StringValue(zones[z].Name), z, err)
+				client := p.clients[zones[z].profile]
+				if _, err := client.ChangeResourceRecordSetsWithContext(ctx, params); err != nil {
+					log.Errorf("Failure in zone %s [Id: %s] when submitting change batch: %v", aws.StringValue(zones[z].zone.Name), z, err)
```

Comment:
Wdyt about adding profile used in this log ?

Reply:
Fully agree. Added some extra log fields in the whole loop over the zones.

Reply:
\o that's a good idea. It would be out of scope but it would definitely help on this code.

Reply:
Definitely going to do that once this PR is merge.